src/components/templates/PartyMap/components/Map/MapRoom.scss

```diff
@@ -13,21 +13,25 @@ $label-visible: flex;
 
   transition: transform 600ms cubic-bezier(0.23, 1, 0.32, 1);
 
-  &:hover {
-    z-index: z(map-room-hovered);
-    transform: scale(1.1);
+  &--active {
```

Comment:
Perhaps could call this portal or similar? Or maybe leave it not as a variant since it'll likely be the main/default type?

The specific reason I wanted to comment here though was that the styles for the other more specific variants such as `--always-show-label` will likely only make sense on this 'default type' as well, as the 'unclickable' type definitely won't need them, etc.